packages/plugin-ext/src/plugin/tests.ts

```diff
@@ -335,6 +336,8 @@ export class TestingExtImpl implements TestingExt {
                     return this.toTestItem(arg);
                 } else if (Array.isArray(arg)) {
                     return arg.map(param => TestItemReference.is(param) ? this.toTestItem(param) : param);
```

Comment:
The argument processor is invoked for each argument we pass to the command, so if we pass the arguments `[testItemReference, testMessageDto]`, we just need to add code to transform TestMessageDto (if necessary).